assets/ais/styles/aisCommon.css

```diff
@@ -2628,6 +2635,10 @@ div.HealthQuestionnaireEditor td.QuestionAndAnswer div.Question textarea {
 .CheckboxedConditionItem {
   padding-bottom: 3px;
 }
+/* new form structurem*/
+.form-input input[type=checkbox] {
+  margin: .125rem .5rem .125rem 0;
+}

```

Comment:
This is to deal with issue of details being too close to the checkbox, which was caused by an underlying checkbox style issue